bin/uglifyjs

```diff
@@ -74,6 +74,7 @@ You need to pass an argument to this option to specify the name that your module
     .describe("reserve-domprops", "Make (most?) DOM properties reserved for --mangle-props")
     .describe("mangle-props", "Mangle property names (0 - disabled, 1 - mangle all properties, 2 - mangle unquoted properies)")
     .describe("mangle-regex", "Only mangle property names matching the regex")
+    .describe("mangle-prefix", "Prefix of the mangle properties")
```

Comment:
Please rename this flag to `mangle-props-prefix` in `bin/uglifyjs`. The property name in lib/propmangle.js is fine.